src/dropdown/dropdown.js

```diff
@@ -115,6 +116,8 @@ angular.module('ui.bootstrap.dropdown', ['ui.bootstrap.position'])
         self.dropdownMenu.remove();
       });
     }
+
+    $element.on('keydown', keybindFilter);
```

Comment:
I believe this would need to get cleaned up on scope destroy of the directive if we chose to go with this implementation.